samples/node/slack/README.adoc

```diff
@@ -0,0 +1,148 @@
+= slack-command
+
+[horizontal]
+*Language*:: JavaScript
+*Function*:: https://github.com/projectriff/riff/blob/master/samples/node/slack/slack-command/slack_command.js[slack_command.js]
```

Comment:
Why don't we use relative links? (this applies globally, obviously)

Reply:
agree